From a263b7fa258803a1677e2b449399bd03f0723e8e Mon Sep 17 00:00:00 2001
From: Tobias Dubois <tobias.dubois@sony.com>
Date: Mon, 11 Jun 2018 17:00:47 +0200
Subject: [PATCH] Lint fix: Invalid vector path in horizontal_ellipsis

Use 0.9 instead of .9 in the vector drawable path to
avoid crashes on some devices.

Test: Manual
Change-Id: I9d5889fdf48fbb2f09995b75d4f20c47454ef591
---
 res/drawable/horizontal_ellipsis.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/drawable/horizontal_ellipsis.xml b/res/drawable/horizontal_ellipsis.xml
index ad0852901..65e958d42 100644
--- a/res/drawable/horizontal_ellipsis.xml
+++ b/res/drawable/horizontal_ellipsis.xml
@@ -22,6 +22,6 @@
         android:tint="?android:attr/textColorSecondary" >
 
     <path
-        android:pathData="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
+        android:pathData="M6 10c-1.1 0-2 0.9-2 2s0.9 2 2 2 2-0.9 2-2-0.9-2-2-2zm12 0c-1.1 0-2 0.9-2 2s0.9 2 2 2 2-0.9 2-2-0.9-2-2-2zm-6 0c-1.1 0-2 0.9-2 2s0.9 2 2 2 2-0.9 2-2-0.9-2-2-2z"
         android:fillColor="@android:color/white" />
 </vector>
\ No newline at end of file
-- 
2.17.1

